sdk/storage/azure-storage-file-datalake/src/main/java/com/azure/storage/file/datalake/options/DataLakePathCreateOptions.java

```diff
@@ -16,18 +19,25 @@ public class DataLakePathCreateOptions {
 
     private String permissions;
     private String umask;
+    private List<PathAccessControlEntry> accessControlEntryList;
+    private String owner;
+    private String group;
+    private DataLakePathScheduleDeletionOptions deletionOptions;
     private PathHttpHeaders headers;
     private Map<String, String> metadata;
     private DataLakeRequestConditions requestConditions;
+    private String sourceLeaseId;
+    private String proposedLeaseId;
+    private Integer leaseDuration;
 
     /**
-     * Constructs a {@link DataLakePathCreateOptions}.
+     * Optional parameters for creating a file or directory.
      */
     public DataLakePathCreateOptions() {
     }
 
     /**
-     * Gets the permissions.
+     * Optional and only valid if Hierarchical Namespace is enabled for the account.
```

Comment:
Would somebody try to use datalake sdk with non-hierarchical namespace ? I'm not sure we need this wording here (and other docs).

Reply:
Flat namespace is not supported by the Data Lake SDK, I think we can safely remove this comment.